code/game/objects/mail.dm

```diff
@@ -0,0 +1,233 @@
+	/obj/item/reagent_containers/food/drinks/bottle/absinthe/premium,
+	/obj/item/clothing/glasses/meson/gar,
+	/obj/item/stack/marker_beacon/ten,
+	/obj/item/stack/medical/splint,
+	/obj/item/pen/multi/fountain,
+	/obj/item/clothing/mask/cigarette/cigar,
+	/obj/item/stack/wrapping_paper,
+	/obj/item/toy/figure/crew/cargotech,
+	/obj/item/toy/figure/crew/qm,
+	/obj/item/toy/figure/crew/miner)
+	job_list = list("Quartermaster", "Cargo Technician", "Shaft Miner")
+
+/obj/item/envelope/medical
+	icon_state = "mail_med"
+	possible_contents = list(/obj/item/soap,
+	/obj/item/reagent_containers/glass/bottle/morphine,
+	/obj/item/reagent_containers/hypospray/safety,
+	/obj/item/reagent_containers/applicator/brute,
+	/obj/item/reagent_containers/applicator/burn,
+	/obj/item/clothing/glasses/sunglasses,
+	/obj/item/reagent_containers/food/snacks/fortunecookie,
+	/obj/item/scalpel/laser/laser1,
+	/obj/item/toy/figure/crew/cmo,
+	/obj/item/toy/figure/crew/chemist,
+	/obj/item/toy/figure/crew/geneticist,
+	/obj/item/toy/figure/crew/md,
+	/obj/item/toy/figure/crew/virologist)
+	job_list = list("Chief Medical Officer", "Medical Doctor", "Coroner", "Chemist", "Virologist", "Psychiatrist", "Psychologist", "Paramedic")
+
+/obj/item/envelope/engineering
+	icon_state = "mail_eng"
+	possible_contents = list(/obj/item/airlock_electronics,
+	/obj/item/reagent_containers/food/drinks/cans/beer,
+	/obj/item/reagent_containers/food/snacks/candy/confectionery/nougat,
+	/obj/item/mod/module/storage/large_capacity,
+	/obj/item/weldingtool/hugetank,
+	/obj/item/geiger_counter,
+	/obj/item/rcd_ammo,
+	/obj/item/grenade/gas/oxygen,
+	/obj/item/toy/figure/crew/atmos,
+	/obj/item/toy/figure/crew/ce,
+	/obj/item/toy/figure/crew/engineer)
+	job_list = list("Chief Engineer", "Station Engineer", "Engine Technician", "Electrician", "Life Support Specialist", "Atmospheric Technician")
+
+/obj/item/envelope/service
+	icon_state = "mail_serv"
+	possible_contents = list(/obj/item/painter,
+	/obj/item/push_broom,
+	/obj/item/gun/energy/floragun,
+	/obj/item/reagent_containers/food/drinks/bottle/fernet,
+	/obj/item/whetstone,
+	/obj/item/reagent_containers/food/drinks/bottle/holywater,
+	/obj/item/stack/ore/tranquillite,
+	/obj/item/stack/ore/bananium,
+	/obj/item/toy/figure/crew/bartender,
+	/obj/item/toy/figure/crew/botanist,
+	/obj/item/toy/figure/crew/chef,
+	/obj/item/toy/figure/crew/clown,
+	/obj/item/toy/figure/crew/hop,
+	/obj/item/toy/figure/crew/chaplain,
+	/obj/item/toy/figure/crew/janitor,
+	/obj/item/toy/figure/crew/librarian,
+	/obj/item/toy/figure/crew/mime)
+	job_list = list("Head of Personnel", "Bartender", "Chef", "Cook", "Culinary Artist", "Butcher", "Botanist", "Hydroponicist", "Botanical Researcher", "Clown", "Mime", "Janitor", "Custodial Technician", "Librarian", "Journalist", "Barber")
+
+/obj/item/envelope/command
+	icon_state = "mail_com"
+	possible_contents = list(/obj/item/flash,
```

Comment:
Could add expensive pens and maybe zippos here, just throwing out ideas.

Reply:
I don't think this is necessary.